proto/money/rpc/master_folio_svc.proto

```diff
@@ -48,10 +49,23 @@ enum MasterFolioSvcFolioSwapResult {
 	MASTER_FOLIO_SVC_FOLIO_SWAP_FAILURE_UNKNOWN = 3;
 }
 
+message MasterFolioSvcReservationMasterFolioAssignmentList {
+	repeated ReservationMasterFolioAssignment assignments = 1;
+}
+
+message ReservationMasterFolioAssignment {
+	.holms.types.booking.indicators.ReservationIndicator client_folio = 1;
```

Comment:
This needs to be a repeated field. A master can have multiple clients. As for displaying that fact...I think we need to talk to Andrew.